### PR TITLE
secound commit

### PR DESCRIPTION
**Pull request by Jaciubek**
fixed paths, removed picture tags, blocked display of appropriate classes depending on the selected display (mobile, tablet, desktop)

### CODE REVIEW

**Comment by Jaciubek** [comment]
fixed paths, removed picture tags, blocked display of appropriate classes depending on the selected display (mobile, tablet, desktop)

File: src/partials/about.html [hunk 1]
```diff
@@ -42,24 +42,21 @@ <h3 class="about-list-header">04</h3>
...
-              ../img/memoji.png 1x,
-              ../img/memoji@2x.png 2x
-            " />
-          <img class="about-picture-img" src="../img/memoji.png" alt="Smiling girl pointing her finger" />
-          <source
-            srcset="
-              ../img/memoji.png 1x,
-              ../img/memoji@2x.png 2x
-            "
+          <img class="about-picture-mobile"
```
Comment (PiotrJasinski1995) [inline src/partials/about.html]: look again on prop-leave-app section:
prop-leave-app-img-html

Comment (PiotrJasinski1995) [reply src/partials/about.html]: why do you add 3 different images? there is only one img

File: src/css/about.css [hunk 2]
```diff
@@ -148,6 +152,14 @@
         height: 243px;
         border-radius: 30px;
     }
+
+    .about-picture-mobile{
```
Comment (PiotrJasinski1995) [inline src/css/about.css]: this styling is wrong, there is only one picture, use media queries with one class (there is only one image!) for proper styling for different screen sizes (please look again how it is done in prop-leave-app-section)

**Review by PiotrJasinski1995 — CHANGES_REQUESTED** [review]
please fix the errors

File: src/css/about.css (hunk 2 above)
Comment (PiotrJasinski1995) [reply src/css/about.css]: what about ::after pseudo element display, it is also not corrected

**Pull request merged**